### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by brytebee**
## In this PR, we accomplished the following:
- Installed webpack
- Installed linters
- Installed Jest
- We implemented GitFlow
- Setup GitHub project with Kanban board

### CODE REVIEW

File: linters.sh [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    "dist/",
+    "build/"
+  ]
+}' >> .eslintrc.json
+echo "Installing Webhint"
+npm install --save-dev hint@6.x
+echo "Installing Stylelint"
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+echo "Installing ESLint"
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
```
Comment (mckent05) [inline linters.sh]: - [**REQUIRED**] Please remove all personal scripts as this is not part of the project requirements.

File: webpack_within_project_file.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+cd src && touch index.js style.css index.html
+cd ..
+echo "Installing Webpack"
+npm install webpack webpack-cli --save-dev
+echo "Installing Style Loader & css-loader"
+npm install --save-dev style-loader css-loader
+echo "Installing Webpack"
+npm install --save-dev html-webpack-plugin
+echo "Installing Webpack-dev-server"
+npm install --save-dev webpack-dev-server
```
Comment (mckent05) [inline webpack_within_project_file.sh]: [**REQUIRED**] As explained previously in your `linters.sh` remove all scripts

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hello @brytebee 👋 

You have done a great job on this project so far 💪 

## Suggested Changes ♻️ 

However, there are still some required changes you need to implement to get this project approved.

_Find below the required/ optional changes in the commented sections._

Ask for another code review implementing the changes

**_Happy coding 👍 👍_**

**Comment by brytebee** [comment]
Hello @name 👋👋,

Thanks for the warm-hearted review 👀👀.
I'll go ahead to implement the requested changes ♻️ and ask for a re-review!

**Review by mckent05 — APPROVED** [review]
Hello @brytebee 👋 

Well done on implementing the previously requested changes 👍 . There's nothing else to say than, it's time to merge it


Congratulations! 🎉 . Cheers and Happy coding: 👏 👏 👏

**Comment by brytebee** [comment]
Hello @mckent05 👋👋,

Thanks for the warm-hearted review 👀👀.
I'll go ahead to merge and move on to the next milestone 🏁🏁.

**_Happy Hacking 😄😄_**

**Pull request merged**